lib/plugins/aws/package/compile/events/eventBridge/index.js

```diff
@@ -147,6 +147,10 @@ class AwsCompileEventBridgeEvents {
                     ],
                   ],
                 };
+                let fullRuleName = `rule/${eventBusName}/${RuleName}`;
```

Comment:
Let's maybe define it with `const` as  `(eventBusName === 'default') ? .. : ...`